integreat_cms/cms/views/utils/search_content_ajax.py

```diff
@@ -71,7 +72,7 @@ def format_object_translation(
 
 
 @require_POST
-def search_content_ajax(
+def search_content_ajax(  # noqa: PLR0915, PLR0912, C901
```

Comment:
I don't think we should just comment these lines out. All three seem reasonable to me. Could you please take the opportunity to refactor this method and split it up into multiple methods, so we don't need these ignores? :)

Reply:
It'll be tackled as #3558 :)

Reply:
#3562 is opened for refactoring. This does not bring a thorough refatoring but should be enough for now.